go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/labstack/echo/v4 v4.11.3
 	github.com/labstack/gommon v0.4.1
 	github.com/prometheus/client_golang v1.19.1
+	github.com/stretchr/testify v1.9.0
```

Comment:
This is a new dependency ([github.com/stretchr/testify](https://github.com/stretchr/testify)) used in [pkg/server/subscriber_test.go](https://github.com/bluesky-social/jetstream/pull/16/files#diff-ce3f3a9c4698b3c6a245360a8526d9f5d789ece4a1d18776b38d653b3fd4435b).

I started using `reflect.DeepEqual` at first, but fell back to this library because I'm familiar with it.